packages/block-library/src/search/style.scss

```
@@ -33,11 +33,11 @@
 }
 
 .wp-block-search__input {
-	padding: 8px;
+	padding: 0.5rem;
```

If we're updating this, given it is meant to match our grid spacing, perhaps we should use the SCSS grid variables instead of hardcoding?
```suggestion
	padding: $grid-unit-10;
```